src/components/ui/UiTrambusLogo.vue

```diff
@@ -0,0 +1,21 @@
+<script lang="ts" setup>
+import trambusLogo from '@/assets/icons/trambusLogo.png'
+</script>
+
+<template>
+  <div class="flex flex-col items-start p-0 gap-2 w-4/5 h-28">
+    <div class="flex flex-row items-start p-0 gap-2 h-14">
+      <div class="h-14 w-14 flex flex-col justify-center">
+        <img :src="trambusLogo" />
```

Comment:
I would create a component for the logo in itself
don't hesitate to create very small component, as it's very easy with Vue, you just need a `<template>` within the file.